packages/shared/src/constants.ts

```diff
@@ -41,14 +41,19 @@ export const AUTOMATIC_DIMENSION_OTHER_NAME = "__Other__";
 
 export const GROWTHBOOK_SECURE_ATTRIBUTE_SALT = "eg8amUur5GunJXCfgjwB";
 
-export const OWNER_ROLES = [
-  "Engineer",
-  "Data Scientist",
-  "Project Manager",
-  "Marketer",
-  "Designer",
-  "Other",
-];
+export enum OwnerJobTitle {
+  engineer = "Engineer",
+  dataScientist = "Data Scientist",
+  projectManager = "Project Manager",
+  marketer = "Marketer",
+  designer = "Designer",
+  other = "Other",
+}
+
+export enum UsageIntent {
+  featureFlags = "Feature Flags",
+  experiments = "Experiments",
+}
```

Comment:
It's usually best practice to avoid Typescript enums.  In this case, exporting an object would work fine and let you derive types from it for both the keys and value, which could be useful, especially if we want to be able to update the display values later without having to change the keys that are stored in the db.

For example:

```ts
const titles = {
  engineer: "Engineer",
  pm: "Project Manager",
} as const;

type TitleKey = keyof typeof titles;       // "engineer" | "pm"
type TitleValue = typeof titles[TitleKey]; // "Engineer" | "Project Manager"
```